inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -265,27 +265,59 @@
 {% endmacro %}
 
 {% macro record_plots(record) %}
+  {% set vars = {'plotExists': False} %}
```

Comment:
Yes - that'd be actually great! I was wondering where this feature went, but eventually found a workaround. I'll refactor this code to work with `do`.